docker-compose.yml

```diff
@@ -14,6 +14,7 @@ services:
         BUNDLER_VERSION: '2.0.2'
     image: '$IMAGE_NAME'
     environment:
+      HOME: '/tmp'
```

Comment:
Am I going to jail for this one?

Reply:
`/dev/null` might be a good reason to go prison :P